inbm/proto/common/v1/common.proto

```diff
@@ -50,12 +51,12 @@ message RepeatedSchedule {
 }
 
 message Operation {
-  repeated PreOperation pre_operations = 1;
-  repeated PostOperation post_operations = 2;
+  repeated PreOperation pre_operations = 1 [(buf.validate.field).repeated = {min_items: 1}];
```

Comment:
min_items should be 0, not 1

Reply:
is it applicable to post_operations too?

Reply:
> is it applicable to post_operations too?

yes -- it's acceptable for both `pre` and `post` to be empty